Lib/email/utils.py

```diff
@@ -106,12 +106,42 @@ def formataddr(pair, charset='utf-8'):
     return address
 
 
+def _pre_parse_validation(fieldvalues):
```

Comment:
the name "fieldvalues" is non-specific.  i realize it comes from the name in getaddress() but we should make it more clear what these are.  email_address_fields perhaps?

Related: I don't this docstring adds meaningful value.  naming the function and parameter right along with it being short code is sufficiently self explanatory for this internal function.  get rid of the docstring.